src/wp-admin/includes/image-edit.php

```diff
@@ -391,6 +397,12 @@ function wp_save_image_file( $filename, $image, $mime_type, $post_id ) {
 				return imagepng( $image, $filename );
 			case 'image/gif':
 				return imagegif( $image, $filename );
+			case 'image/webp':
+				if ( function_exists( 'imagewebp' ) ) {
+					/** This filter is documented in wp-includes/class-wp-image-editor.php */
+					return imagewebp( $image, null, apply_filters( 'jpeg_quality', 90, 'edit_image' ) );
```

Comment:
hmm, wonder if this should rather be `webp_quality` and default to a lower value. Actual jpeg default is 82, I think this could sit at 75 for WebP.

Reply:
See https://make.wordpress.org/core/2016/02/22/proposal-increase-the-default-image-compression-in-wordpress/ for the reasons behind the value of 82. Would be nice to have some similar data for WebP.

Reply:
May also be of use: https://www.industrialempathy.com/posts/avif-webp-quality-settings/

Reply:
Actually, I'm going to remove this, the filter is a legacy filter for jpeg, I think we can let developers rely on the `wp_editor_set_quality` filter.